### ISSUE TITLE
GIS Support User access to data on GISDM / help with ArcMap - J. Bradley

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-data-tech#2147]
Josh emailed asking about adding the ASMP Street Network from AGOL to ArcMap. I let him know that the data is available on GISDM. I asked him to call me if he needs assistance getting it setup and how to access the data.

**Comment by jaime-mckeown** [comment]
- I helped setup Josh with connections to GISDM, GISDM External, and folder connections.
- I worked with him over the phone to find the data he was looking for, then gave him some tips for using ArcMap for what his needs were at this particular time as he was going to manually do a task that would take a few minutes to find using the tools.
- We ended up having a mini-training session to get him more familiar with some of the basic tools in ArcMap.
- He was then going to try and figure out the same steps to get the necessary information for the other 2 engineering service areas to help his co-workers out.
